src/layer/tile/GridLayer.js

```diff
@@ -85,8 +85,10 @@ export var GridLayer = Layer.extend({
 		// Opacity of the tiles. Can be used in the `createTile()` function.
 		opacity: 1,
 
-		// @option updateWhenIdle: Boolean = depends
-		// If `false`, new tiles are loaded during panning, otherwise only after it (for better performance). `true` by default on mobile browsers, otherwise `false`.
+		// @option updateWhenIdle: Boolean = (depends)
+		// Load new tiles only when panning ends.
+		// `true` by default on mobile browsers, in order to avoid too many requests and keep smooth navigation.
+		// `false` otherwise in order to display new tiles during panning, which can be extensive on desktop browsers.
```

Comment:
Lol, thanks, now I understand the context! I think your suggestion is good, I would make some minor adjustments, how about:

"`false` otherwise, in order to display new tiles during panning, since it is easy to pan outside the `keepBuffer` in desktop browsers."